docs/sources/administration/api-keys/create-api-key.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Create a API key in Grafana
+menuTitle: Create a API key
+aliases: [docs/sources/administration/service-accounts/create-api-key.md]
+description: 'How to create a API key in Grafana'
+weight: 50
+keywords:
+  - API keys
+  - Service accounts
+---
+
+# Create a API key in Grafana
+
+You can create a API key in the API keys tab of the organization. To manage your computed workload with a user that is not tied to a user specifically.
```

Comment:
```suggestion
You can create an API key in the API keys tab of the organization. To manage your computed workload with a user that is not tied to a user specifically.
```

I don't understand the second sentence though? It seems incomplete

Reply:
maybe the second sentence makes more sense when e use the same sentence that we use for service accounts, and really skip the whole (not tied to a user).

The key can be used to run automated or compute workloads for specific use cases.